common/display/displayplanemanager.cpp

```diff
@@ -238,16 +238,18 @@ bool DisplayPlaneManager::ValidateLayers(
 
           // If we are able to composite buffer with the given plane, lets use
           // it.
-          bool fall_back = FallbacktoGPU(plane, layer, composition);
-          test_commit_done = true;
-          if (fall_back) {
-            ISURFACETRACE(
-                "Force GPU rander the plane[%d], for the layer[%d] isVideo: "
-                "%d, isSolidColor: %d, alpha: %d",
-                last_plane.GetDisplayPlane()->id(), layer->GetZorder(),
-                layer->IsVideoLayer(), layer->IsSolidColor(),
-                layer->GetAlpha());
-            last_plane.ForceGPURendering();
+          if(plane != NULL) {
+            bool fall_back = FallbacktoGPU(plane, layer, composition);
```

Comment:
The scan tool only worry about if plane is null in invoking FallbacktoGPU right? if so, only check null before invoking this method. no need to check for the reset